src/app.js

```diff
@@ -43,14 +54,30 @@ passport.use(
   )
 );
 
+// Set up passport to use local config
+passport.use(new LocalStrategy({ usernameField: 'email' }, localConfig));
+passport.serializeUser((user, done) => {
+  done(null, user.uid);
+});
+passport.deserializeUser((uid, done) => {
+  // In the future, we might store user info (name, roles, etc) in Mongo via the passport.authenticate cb.
```

Comment:
super minor, but the reference to Mongo doesn't really make sense since we aren't using mongo. plus the line length warning

Reply:
Removed mongo but left the comment